include/bus_handler.hpp

```diff
@@ -116,6 +120,13 @@ class BusHandler
      * @param[in] event: Button event
      */
     void btnRequest(int event);
+
+    /**
+     * @brief API to trigger specific panel functions.
+     *
+     * @param[in] funcNum - Function number to execute
+     */
+    void triggerPanelFunc(const types::FunctionNumber funcNum);
```

Comment:
It can be used for all functions that are valid to be executed, so may be a check or valid supported functions in description will be good.

Reply:
done